docs/nulecule.md

```diff
@@ -0,0 +1,242 @@
+# Nulecule file
```

Comment:
This file is awesome.. I worry about rot though - if the specification ever starts changing again.